integration/commands_freemonitoring_test.go

```diff
@@ -32,28 +31,22 @@ func TestCommandsFreeMonitoringGetFreeMonitoringStatus(t *testing.T) {
 		DatabaseName: "admin",
 	})
 
-	expected := map[string]any{
-		"state": "disabled",
-		"ok":    float64(1),
-	}
+	necessaryKeys := []string{"state", "ok"}
+
+	allowedKeys := []string{"state", "message", "url", "userReminder", "ok"}
 
 	var actual bson.D
 	err := s.Collection.Database().RunCommand(s.Ctx, bson.D{{"getFreeMonitoringStatus", 1}}).Decode(&actual)
 	require.NoError(t, err)
 
-	m := actual.Map()
 	keys := CollectKeys(t, actual)
 
-	for k, item := range expected {
+	for _, k := range allowedKeys {
+		assert.Contains(t, allowedKeys, k)
```

Comment:
Any reason why we don't check the value of each key anymore? Perhaps it's covered by other tests :thinking:

Reply:
MongoDB will respond with a different output. They will use the userReminder and url fields to notify about some FreeMonitoring specific things. The expected output for both handlers is fully covered inside the dance tests PR

Reply:
Great thanks for the explanation!